modules/002-deckhouse/templates/common/rbac/kube-rbac-proxy.yaml

```diff
@@ -11,3 +11,6 @@ rules:
 - apiGroups: ["authorization.k8s.io"]
   resources: ["subjectaccessreviews"]
   verbs: ["create"]
+- apiGroups: ["discovery.k8s.io"]
```

Comment:
Why do kube-rbac-proxy need the access to endpointslices?

Reply:
The node-local-dns Pods need it. So, configure rbac-for-us.yaml in node-local-dns module.